docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md

```diff
@@ -112,365 +112,590 @@ By providing a standardized, scalable and easy-to-adopt solution, the EcoPass KI
-Accordingly, the Battery Pass establishes the fundamental framework for digital infrastructures aimed at documenting and facilitating the exchange of essential information and update-relevant technical data. Specifically, it focuses on data that provides a comprehensive account of supply chain responsibility, e.g. the carbon footprint, working conditions during raw material extraction and the assessment of battery conditions. It is structured according to the following categories:
+### Digital Product Passport
 
-- Circularity
-- Metadata
-- Identification
-- Performance
-- Sources
-- Physical Dimension
-- Safety
-- General Information
-- Conformity
-- Chemical Material
-  In addition, the Battery Pass data model offers the following advantages:
-- It is self-describing, which means it has all the details necessary to comprehend what it includes. Different systems can easily interpret the Battery Pass as a result.
-- It is extendable, therefore new information can be added without much difficulty. As a result, the Battery Pass will be able to adapt to the changing requirements of the battery sector.
-- It establishes a common semantic in Catena-X which other passes can build upon.
+#### Digital Product Passport Introduction
 
-#### Link to current Battery Pass model
+Digital Product Passports addresses three needs by the industry: First, they gather all relevant information about a product in a single, digital tool. Second, digital product passports answer increasing regulatory requirements and demand for sustainability information along the value chain of a product. And third, Digital Product Passports are an important enabler for a circular economy.
+The collected data is defined via standards specified by the Catena-X network and obtained from the industry. The content of the Product Passport is product-related information that is useful and necessary for the consumer, and other actors who will deal with the product till end of its life, and is limited to the required minimum of the necessary information.
+The data model in version 4.0.0 corresponds to the Digital Product Passport information required by the proposed Ecodesign Regulation ([ESPR-2022](<(https://commission.europa.eu/energy-climate-change-environment/standards-tools-and-labels/products-labelling-rules-and-requirements/sustainable-products/ecodesign-sustainable-products_en)>)) from March 30th, 2022. In the future, delegated acts for specific product groups would require a product passport to be available for each product. The data model will be updated, as newer versions of the regulation will be published. The latest version of the document was the [provisional agreement](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CONSIL:ST_5147_2024_INIT) between the EU Council and the Parliament from January 9th, 2024. The text is informal, but the content of the final regulation was agreed between these two institutions.
```

Comment:
Please verify that this is still the latest version of published regulation - there is a new delegated act published by the EC on EV batteries.

Reply:
This is the regulation about  Digital Product Passports (ESPR = Ecodesign for Sustainable Products Regulation). This data model correcponds to the indicated version of the document.

Reply:
Thanks for clarifying!